packages/components/tabs/src/Tab.tsx

```diff
@@ -4,13 +4,16 @@ import type { CommonProps } from '@contentful/f36-core';
 import { Box } from '@contentful/f36-core';
 
 import { getTabStyles } from './Tabs.styles';
-
+import { useTabsContext } from './tabsContext';
 export interface TabProps extends CommonProps {
-  id: string;
+  /**
+   * Takes id of the TabPanel it panelId
```

Comment:
```suggestion
   * Takes id of the TabPanel it controls
```

Did you mean that?

Maybe you did a replace all `control` to `panelId` hahaha

Reply:
good catch, ahaha